README.md

```diff
@@ -1,9 +1,9 @@
-# Push Notifications Android 
+# Beams Android SDK
```

Comment:
Maybe for a transitional period, we want to go with `Beams Push Notifications SDK`? What do you think?

Reply:
Maybe this doesn't apply anymore @jonathanlloyd.